README.md

```diff
@@ -151,21 +151,22 @@ The SECC and EVCC have been tested together under:
 ## IPv6 WARNING
 
 For the system to work locally, the network interface needs to have
-an IPv6 link-local address assigned.
+an IPv6 local-link address assigned.
```

Comment:
A quick Google search suggested that "link-local" is the correct one -- @tropxy are you sure about this?  According to Wikipedia, "In computer networking, a link-local address is a network address that is valid only for communications within the network segment or the broadcast domain that the host is connected to."

Reply:
you are right, it is link-local